src/action/libc_version.c

```diff
@@ -0,0 +1,14 @@
+#include <features.h>
+#ifdef __GNU_LIBRARY__
+#include <gnu/libc-version.h>
+#else
+#include <stddef.h>
+#endif
+
+const char *libc_version(void) {
+#ifdef __GNU_LIBRARY__
+  return gnu_get_libc_version();
+#else
+  return NULL;
+#endif
+}
```

Comment:
What is the benefit over [`libc::gnu_get_libc_version`]?

[`libc::gnu_get_libc_version`]: https://docs.rs/libc/0.2.137/libc/fn.gnu_get_libc_version.html

Reply:
Yes, excellent. I did not know that the gnu_get_* APIs were added recently

https://github.com/rust-lang/libc/commit/f4150780462d20d709369743bbc7f7a9e26cb6ca